activerecord/test/cases/associations/inverse_associations_test.rb

```diff
@@ -729,6 +729,18 @@ def test_inversed_instance_should_not_be_reloaded_after_stale_state_changed_with
     assert_same old_inversed_man, new_inversed_man
   end
 
+  def test_inverse_instance_should_be_loaded_after_autosave_if_it_is_not_already_loaded
+    man = Man.create!
+    man.create_autosave_face!
+
+    man.autosave_face.reload # clear cached load of autosave_man
+    man.autosave_face.description = "new description"
+
+    assert_equal(false, Man.find(man.id).autosave_face.autosave_man.nil?)
+    man.save!
+    assert_equal(false, man.autosave_face.autosave_man.nil?)
```

Comment:
```suggestion
    man.save!

    assert_not_nil man.autosave_face.autosave_man
```

I don't think we need to assert that the inverse association works correctly before the record is saved.